compiled_starters/git-starter-kotlin/your_git.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
+
+tmpFile=$(mktemp tmp.XXXXXXXXXX.jar)
+kotlinc "$(dirname $0)/app/main.kt" -include-runtime -d "$tmpFile"
+kotlin -classpath "$tmpFile" MainKt "$@"
+
+rm "$tmpFile"
```

Comment:
To ensure that signals are passed properly to the child process, we'll need to make the last command here `kotlin`, and prepend `exec` to it. 

Example: https://github.com/codecrafters-io/languages/blob/5746db439eae64c99189ce464509f0d4a747f2b1/compiled_starters/git-starter-ruby/your_git.sh#L2

(Just realizing that this is something I could add to the contributing guide 🙂) 

And since we won't be able to remove the tempfile with this strategy, maybe change it to just `mktemp` so that the file isn't created in the repository directory but in a temporary location instead.

Reply:
A bit of context on where this signal forwarding is used: 

All tests run with a specific timeout, most commonly 5 seconds, but sometimes 30s (if it is expected to make outward HTTP requests). When this timeout is hit, we send `SIGTERM` to the process to terminate it. Also, in certain challenges like Redis Redis, we terminate the program once we've finished running tests against it - that uses signals too.

Reply:
Ah, good point. Thanks for providing the context for where this is used. I've made a patch to correct this.